vars/rapidastUtils.groovy

```diff
@@ -24,6 +24,35 @@ def prepareRapidastStages(String ServiceName, String ApiScanner, String TargetUr
         stage("Collect artifacts") {
             archiveArtifacts allowEmptyArchive: true, artifacts: "results/${ServiceName}/**/zap/*.*, , results.html"
         }
+
+        stage("Send data to Sitreps Grafana") {
+            // There is a a dir that contains a timestamp which would be harder to predict, instead try to find resource.
+            def RAPIDAST_JSON_REPORT = sh returnStdout: true, script: "find . -name zap-report.json"
+            String[] str = RAPIDAST_JSON_REPORT.split("/");
+            def random_results_dir = str[3]
+            def html = "${BUILD_URL}/artifact/results/${ServiceName}/${random_results_dir}/zap/zap-report.html"
+            def json_path = "results/${ServiceName}/${random_results_dir}/zap/zap-report.json"
+            def raw_json = readJSON file: json_path
+            def data = [
+              service: "${ServiceName}",
+              report: raw_json,
+              html_url: html
+            ]
+            def jsonData = groovy.json.JsonOutput.toJson(data)
+            def apiUrl = "https://sitreps-backend-insights-qe-sitreps.apps.ocp-c1.prod.psi.redhat.com/api/v1/rapidast/"
```

Comment:
Let's add this in https://github.com/RedHatInsights/insights-pipeline-lib/blob/master/vars/pipelineVars.groovy